File: src/app/shared/models/Document.ts
export interface Document {
    Id?: string;

    DocumentId?: string | null;

    Name?: string | null;

    Description?: string | null;

    OwnersEmail?: string | null;

    ThumbnailLink?: string | null;

    WebcontentLink?: string | null;

    WebviewLink?: string | null;

    FileExtension?: string | null;

    FullFileExtension?: string | null;

    Kind?: string | null;

    Md5Checksum?: string | null;

    Shared?: boolean;

    Size?: number;

    Version?: number;

    DocumentCreatedTime?: string | null;

    UserId?: string | null;

    CreatedOn?: string;

    CreatorId?: string | null;

    UpdatedOn?: string;

    UpdaterId?: string | null;
}

File: UpdateData/Models/Document.cs
namespace UpdateData.Models;

public class Document:AuditableEntity
{
    public Guid Id { get; set; }
    
    public string DocumentId { get; set; }

    public string Name { get; set; }

    public string Description { get; set; }

    public string OwnersEmail { get; set; }

    public string ThumbnailLink { get; set; }

    public string WebcontentLink { get; set; }

    public string WebviewLink { get; set; }

    public string FileExtension { get; set; }

    public string FullFileExtension { get; set; }

    public string Kind { get; set; }

    public string Md5Checksum { get; set; }

    public bool? Shared { get; set; }

    public long? Size { get; set; }

    public long? Version { get; set; }

    public DateTime? DocumentCreatedTime { get; set; }
    
    public string UserId { get; set; }
}